packages/mdc-menu/mdc-menu.scss

```diff
@@ -118,38 +114,20 @@ $mdc-menu-scale-duration: .12s;
   .mdc-list-item {
     @include mdc-typography(subheading2);
 
-    @include mdc-theme-dark(".mdc-menu", true) {
-      color: white;
-    }
-
     position: relative;
     outline: none;
     color: inherit;
     text-decoration: none;
     user-select: none;
   }
 
-  .mdc-list-divider {
-    @include mdc-theme-dark(".mdc-menu", true) {
-      border-color: rgba(white, .12);
-    }
-  }
-
   .mdc-list-item__graphic {
     color: rgba(black, .54);
```

Comment:
Replace this line with:

`@include mdc-theme-prop(color, text-secondary-on-background);`

Reply:
I traced it back through mdc-theme, so it should be the same, but we don't have a demo to showcase this. Should we add a menu item with a graphic?